### PR TITLE
revised SDS.6,7,8,9,10,12,13,14,15 based on approved corresponding SCS test cases

### PR DESCRIPTION
**Pull request by jni2000**
Federated Wireless revised SDS.6,7,8,9,10,12,13,14,15 based on the approved corresponding SCS..6,7,8,9,10,12,13,14,15 test cases. Please review and give your comments.

### CODE REVIEW

File: src/harness/certs/README.md [hunk 1]
```diff
@@ -12,19 +12,24 @@ WINNF-15-S-0065" document. Naming and base configuration are issued from
...
+             /    |          \              	  	            \
+            /     |           \              	  	             \
+ admin_client  server  client|device_[a|c]|corrupted_client      domain_proxy
+			wrong_type_client|client_expired         corrupted_domain_proxy
+                        client_inapplicable                      wrong_type_domain_proxy
+                                                                 domain_proxy_expired
+                                                                 domain_proxy_inapplicable
+
+
+   unrecognized_ca       non_cbrs_root_ca----------------------
```
Comment (norbertbarbosa) [inline src/harness/certs/README.md]: `unrecognized_ca` is currently named `unrecognized_root_ca`.
Also add the `unrecognized_domain_proxy` under `unrecognized_device`, here in the schema and below in the description

Comment (jni2000) [reply src/harness/certs/README.md]: thanks for the findings, changed accordingly. Please take a look again.

**Comment by jni2000** [comment]
Norbert

Revised README to address your findings. Thanks. Please take a look.

File: src/harness/certs/generate_fake_certs.sh [hunk 2]
```diff
@@ -256,13 +235,86 @@ openssl ca -cert cbsd_ca.cert -keyfile private/cbsd_ca.key -in client_expired.cs
...
+    -subj "/C=US/ST=District of Columbia/L=Washington/O=Wireless Innovation Forum/OU=www.wirelessinnovation.org/CN=Domain Proxy Unknown"
+openssl ca -cert non_cbrs_root_signed_oper_ca.cert -keyfile private/non_cbrs_root_signed_oper_ca.key -in non_cbrs_signed_domain_proxy.csr \
+    -out non_cbrs_signed_domain_proxy.cert -outdir ./root \
+    -policy policy_anything -extensions oper_req_sign -config ../../../cert/openssl.cnf \
+    -batch -notext -create_serial -utf8 -days 1185 -md sha384
+
+# Certificate for test case WINNF.FT.S.SDS.10 - Certificate of wrong type presented during registration
+# Creating a wrong type certificate by reusing the server.csr and having it signed by proxy_ca.
+echo "\n\nGenerate wrong type certificate/key"
+openssl ca -cert proxy_ca.cert -keyfile private/proxy_ca.key -in server.csr \
```
Comment (jcereced) [inline src/harness/certs/generate_fake_certs.sh]: The way the test case is written the certificate is signed by cbsd_sa, so wouldn’t the SAS then accept the certificate and not fail to establish the TLS session?  Should the certificate be signed by the sas_ca instead in this TC?

Comment (jni2000) [reply src/harness/certs/generate_fake_certs.sh]: Agreed with you according to teh following statement. Will make the change.

"Replace the DP Test Harness certificate with an invalid certificate. This certificate shall be valid except for being a certificate of the wrong type but having correct signatures descending from the CBRS trust root (e.g. a SAS certificate, not a CBSD certificate)"

**Comment by jni2000** [comment]
I am going to merge this approved PR in first and then raise a separate PR to adjust SCS.10 and SDS.10 to address Javier's comment.

**Comment by jcereced** [comment]
sounds good!

**Pull request merged**